modules/simple-payments/paypal-express-checkout.js

```diff
@@ -89,6 +90,25 @@ var PaypalExpressCheckout = {
 		PaypalExpressCheckout.showMessage( message, buttonDomId, true );
 	},
 
+	processErrorMessage: function( errorResponse ) {
+		var error = errorResponse.responseJSON;
+		var defaultMessage = 'There was an issue processing your payment.';
+
+		if ( ! error ) {
+			return defaultMessage;
+		}
+
+		if ( error.additional_errors ) {
+			var messages = [];
+			error.additional_errors.forEach( function( error ) {
+				messages.push( '<p>' + error.message.toString() + '</p>' );
```

Comment:
We should probably guard against undefined `message` since that will error if we call `toString()` on that.